staker/errors.gno

```diff
@@ -0,0 +1,32 @@
+package staker
+
+import (
+	"errors"
+
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	errNoPermission                 = errors.New("[GNOSWAP-STAKER-001] no permission")
+	errNotRegistered                = errors.New("[GNOSWAP-STAKER-002] not registered")
+	errAlreadyRegistered            = errors.New("[GNOSWAP-STAKER-003] already registered")
+	errWrapUnwrap                   = errors.New("[GNOSWAP-STAKER-004] wrap/unwrap")
```

Comment:
On what occasion will this occur? Just curious

Reply:
current `wugnot` has [minimum amount](https://github.com/gnolang/gno/blob/1a57e81f5e454304dd0de38657d680edef18100d/examples/gno.land/r/demo/wugnot/wugnot.gno#L18-L20) to wrap/unwrap. 004 error is for that.